nx-X11/programs/Xserver/hw/nxagent/Display.c

```diff
@@ -2842,8 +2847,12 @@ Bool nxagentReconnectDisplay(void *p0)
                                      CopyFromParent,
                                      0L, NULL);
 
+  if (nxagentReportWindowIds) {
+    fprintf(stderr, "NXAGENT_WINDOW_ID: CONFINE,WID:[0x%x]\n",
```

Comment:
Not sure if I get this right, but in line 1339 you use "CONFINEMENT_WINDOW" while here you are using "CONFINE".